### ISSUE TITLE
Nicklist bar not scrolling with mouse when mouse support is enabled

### ISSUE THREAD

**Issue by undeadrevo** [weechat/weechat#1243]
## Bug summary
So, I have a nicklist in the usual spot on the right of buffers of channels. No idea why, but suddenly the mouse will not scroll it. Nothing was changed in the settings between setting up my setup and now(which has been roughly 6 months).

## Current behavior
Nicklist bar doesn't scroll when there is stuff to scroll to.

## Steps to reproduce

1. Have a nicklist bar setup with mouse support turned on.
2. Try to scroll when the cursor is hovering over the nicklist

## Desired behavior
Nicklist bar should scroll when the cursor is hovering over 

## Suggested solutions
No clue

## Additional information

---

- WeeChat version: 2.2
- OS, distribution and version: Arch Linux

**Comment by undeadrevo** [comment]
I know. It showed stuff for the buflist and text output for mouse debug and I did a bit of mucking about(thought I put up a response 2 days ago saying what I did), but I took the mouse button area from a fresh conf and put it in my old one. Turns out my xrandr script may have messed with stuff as well in terms of area recognition. But it works now and has been for the last 2 days(which is how long I intended to wait to see if the fix was permanent or not).
